### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dorbin**
Some minor edits to the README

### CODE REVIEW

**Comment by dorbin** [comment]
@joonlim

File: gke-deploy/README.md [hunk 1]
```diff
@@ -14,35 +14,38 @@ the following:
...
 
 This command will execute only the [Prepare](#prepare-step) phase mentioned
 above.
 
-### [gke-deploy apply [flags]](docs/apply.md)
+`[gke-deploy apply [flags]](docs/apply.md)`
 
 This command will execute only the [Apply](#apply-step) phase mentioned above.
 
 ### [Automated Deployments with GCB](docs/automated-deployments.md)
```
Comment (joonlim) [inline gke-deploy/README.md]: What do you think about changing ### to ## and moving this out of the Usage section?

File: gke-deploy/README.md [hunk 2]
```diff
@@ -14,35 +14,38 @@ the following:
...
-3.  `gke-deploy` waits for applied Kubernetes resources to be ready.
+2. Applies the set of Kubernetes resource YAML configs that were
+modified in the prepare step
+
+3. Waits for applied Kubernetes resources to be ready
 
 ## Usage
 
-### [gke-deploy run [flags]](docs/run.md)
+`[gke-deploy run [flags]](docs/run.md)`
```
Comment (joonlim) [inline gke-deploy/README.md]: [`gke-deploy run [flags]`](docs/run.md)

File: gke-deploy/README.md [hunk 3]
```diff
@@ -14,35 +14,38 @@ the following:
...
 
 This command will execute both the [Prepare](#prepare-step) and
 [Apply](#apply-step) phases mentioned above.
 
-### [gke-deploy prepare [flags]](docs/prepare.md)
+`[gke-deploy prepare [flags]](docs/prepare.md)`
```
Comment (joonlim) [inline gke-deploy/README.md]: [gke-deploy prepare [flags]](docs/run.md)

File: gke-deploy/README.md [hunk 4]
```diff
@@ -14,35 +14,38 @@ the following:
...
 
 This command will execute only the [Prepare](#prepare-step) phase mentioned
 above.
 
-### [gke-deploy apply [flags]](docs/apply.md)
+`[gke-deploy apply [flags]](docs/apply.md)`
```
Comment (joonlim) [inline gke-deploy/README.md]: [`gke-deploy apply [flags]`](docs/apply.md)

File: gke-deploy/README.md (hunk 2 above)
Comment (joonlim) [reply gke-deploy/README.md]: Backticks should be inside braces

**Comment by joonlim** [comment]
@dorbin Have you reviewed the files in `gke-deploy/docs/` as well?

**Comment by joonlim** [comment]
@dorbin could you also add the changes proposed here please: https://github.com/GoogleCloudPlatform/cloud-builders/pull/516/files

**Comment by joonlim** [comment]
@dorbin looks like the above PR I mentioned has already been merged.

**Comment by joonlim** [comment]
@dorbin I've implemented your suggested fixes here: https://github.com/GoogleCloudPlatform/cloud-builders/pull/519.

You can delete this PR.

**Comment by ImJasonH** [comment]
Is this PR still necessary?

**Pull request closed without merging**